code-challenges.js

```diff
@@ -1,10 +1,27 @@
 // ASSESSMENT 3: Coding Practical Questions
 
-
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-
-
+// function will take in a number 10 times or takes in a single number and keeps reiterating it?
+// each number will be added to the last
+// will return an mapped array
+// 0 + 1 = 1, 1+1 = 2
+
+// function will take in a num
+const sequence = (nums) => {
+  // let array = array of 1, 1
+  let fibArray = [1, 1]
+  //   loop starting at 1 and stops just before whatever nums -1 and reiterates by 1 every time
+  for (let i = 1; i < nums - 1; i++) {
+    // arryay is pushed out as two arrays added togther
+    // 1        +       1
+    fibArray.push(fibArray[i] + fibArray[i - 1])
+  }
+  return fibArray
+}
+// [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+// console.log(sequence(fibArray))
+console.log(sequence(10))
```

Comment:
nice work! I like that this one can dynamically return as many pieces of the fibonacci sequence as we want